CBLClient/Apps/CBLTestServer-Android/app/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.application'
 
 ext {
-    DEF_VERSION = "2.8.0-204"
+    DEF_VERSION = "2.8.0-267"
```

Comment:
revert this change, we do need to add latest version here

Reply:
done